src/app/learn/get-started/prereqs/ssl-certificate/index.html

```diff
@@ -104,18 +120,23 @@
   </ol>
 
   <stache-page-anchor>
-    Installation for Windows
+    Manual installation for Windows
   </stache-page-anchor>
 
   <ol>
+    <li>
+      <sky-code-block>
```

Comment:
Same note above about the clunkiness of the code block in the step. It seems like it should just be a step that says "Run `skyux certs generate`."

Reply:
Agreed.